### ISSUE TITLE
replace audio in video

### ISSUE THREAD

**Issue by julioVillagomez** [pascalbaljetmedia/laravel-ffmpeg#166]
How can I replace the audio of a video?

**Comment by pascalbaljet** [comment]
I've rewritten the package and it has support for multiple inputs (complex filters and mapping). Could you test the `v7` branch? Almost no breaking changes except for some filter stuff in HLS exports.

https://github.com/pascalbaljetmedia/laravel-ffmpeg/tree/v7

`composer require "pbmedia/laravel-ffmpeg:v7.x-dev"`

```php
<?php

FFMpeg::fromDisk('local')
    ->open(['video.mp4', 'audio.m4a'])
    ->export()
    ->addFormatOutputMapping(new X264, Media::make('local', 'new_video.mp4'), ['0:v', '1:a'])
    ->save();
````

**Comment by pascalbaljet** [comment]
Fixed in `v7.0`
